pkg/skaffold/deploy/kubectl.go

```diff
@@ -208,6 +208,8 @@ func (k *KubectlDeployer) readManifests() (manifestList, error) {
 		manifests = append(manifests, manifest)
 	}
 
+	logrus.Debugln(manifests.String())
```

Comment:
Nit: could you add a bit of context to these log statements so they're more useful later?